### DISCUSSION TITLE
Build for Neo4J failing

### DISCUSSION THREAD

**Discussion by vayu068** [Sunbird-Ed/Community#102] [Installation] [answered]
Using release tag: release-4.5.0_RC12

Please find below complete Jenkins logs

Started by user Pawan Verma
Checking out git https://github.com/project-sunbird/sunbird-learning-platform.git into /var/lib/jenkins/workspace/Build/KnowledgePlatform/Neo4j@script/a1bebf3b647e12e691744858777e40ddbbbb9667c53cbc0316fe08f66333793b to read build/neo4j-extensions/Jenkinsfile
The recommended git tool is: NONE
No credentials specified
Cloning the remote Git repository
Cloning repository https://github.com/project-sunbird/sunbird-learning-platform.git
 > git init /var/lib/jenkins/workspace/Build/KnowledgePlatform/Neo4j@script/a1bebf3b647e12e691744858777e40ddbbbb9667c53cbc0316fe08f66333793b # timeout=10
Fetching upstream changes from https://github.com/project-sunbird/sunbird-learning-platform.git
 > git --version # timeout=10
 > git --version # 'git version 2.34.1'
 > git fetch --tags --force --progress -- https://github.com/project-sunbird/sunbird-learning-platform.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/project-sunbird/sunbird-learning-platform.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
 > git rev-parse origin/release-4.5.0_RC12^{commit} # timeout=10
 > git rev-parse release-4.5.0_RC12^{commit} # timeout=10
Checking out Revision 836edd7721ba7791438ba5acf3147360093544ee (release-4.5.0_RC12)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 836edd7721ba7791438ba5acf3147360093544ee # timeout=10
Commit message: "Merge pull request #1634 from AmiableAnil/release-4.5.0"
First time build. Skipping changelog.
ERROR: /var/lib/jenkins/workspace/Build/KnowledgePlatform/Neo4j@script/a1bebf3b647e12e691744858777e40ddbbbb9667c53cbc0316fe08f66333793b/build/neo4j-extensions/Jenkinsfile not found
Finished: FAILURE

**Comment by sharathkashyap** [comment, marked as answer]
@keshavprasadms @AmiableAnil Need your inputs into this.

**Reply by beepdot** [reply]
The repo and Jenkinsfile have been moved to another location. Please update your job configuration to point to this repo https://github.com/project-sunbird/knowledge-platform-db-extension
Please use tag as `release-4.1.0_RC2`

**Reply by daveanvit** [reply]
https://github.com/project-sunbird/knowledge-platform-db-extension getting 404 on this @keshavprasadms is it up?

**Reply by beepdot** [reply]
https://github.com/project-sunbird/knowledge-platform-db-extensions

**Reply by daveanvit** [reply]
Thanks @keshavprasadms
